documentation/src/main/asciidoc/topics/proc_configuring_pom_xml.adoc

```diff
@@ -0,0 +1,49 @@
+[id='configure_pom']
+= Configuring Your {brandname} POM
+Maven uses configuration files called Project Object Model (POM) files to
+define projects and manage builds. POM files are in XML format and describe the
+module and component dependencies, build order, and targets for the resulting
+project packaging and output.
+
+.Procedure
+
+. Open your project `pom.xml` for editing.
+. Define the `version.infinispan` property with the correct {brandname} version.
+. Include the `infinispan-bom` in a `dependencyManagement` section.
++
+The Bill Of Materials (BOM) controls dependency versions, which avoids version
+conflicts and means you do not need to set the version for each {brandname}
+artifact you add as a dependency to your project.
++
+. Add {brandname} artifacts in a `dependencies` section as required.
+. Save and close `pom.xml`.
+
+The following example shows the {brandname} version, BOM, and dependencies:
+
+[source,xml,options="nowrap",subs=attributes+]
+----
+<properties>
+  <version.infinispan>{mvn_version}</version.infinispan>
+</properties>
+
+<!-- Use the BOM to manage dependency versions. -->
+<dependencyManagement>
+  <dependencies>
+    <dependency>
+      <groupId>org.infinispan</groupId>
+      <artifactId>infinispan-bom</artifactId>
+      <version>${version.infinispan}</version>
+      <type>pom</type>
+      <scope>import</scope>
+    </dependency>
+  </dependencies>
+</dependencyManagement>
+
+<!-- Add dependencies as required. -->
+<dependencies>
```

Comment:
this can be removed IMO.
this is shown in the Hot Rod client docs as well and it isn't required. 
If you update `ispn_bom.xml`, it can be used here too.

Reply:
do you mean remove the `dependencies` section?

while I know the bom is optional I think it should be recommended. this PR is intended for the users who would rather open a support ticket to get the version details than spend 5 mins to find it. perhaps I don't understand your meaning clearly...

Reply:
> do you mean remove the `dependencies` section?
> 
yes.